src/codeflare_sdk/cluster/cluster.py

```diff
@@ -276,7 +339,7 @@ def wait_ready(self, timeout: Optional[int] = None, dashboard_check: bool = True
         time = 0
         while not ready:
             status, ready = self.status(print_to_console=False)
-            if status == CodeFlareClusterStatus.UNKNOWN:
+            if self.config.mcad and status == CodeFlareClusterStatus.UNKNOWN:
```

Comment:
Inside the conditional on L300 you could remove the check for `cluster.status == RayClusterStatus.UNKNOWN` and set `status` equal to starting that way this logging can still be used in the case where the Ray cluster doesn't exist at all.